app/src/globalConfig.ts

```diff
@@ -108,3 +108,10 @@ export const removeMediaProgress = (mediaId: string, contentId: Uuid) => {
         localStorage.setItem("mediaProgress", JSON.stringify(_mediaProgress));
     }
 };
+
+/**
+ * The user preferences.
+ */
+export const userPreferences = ref<{ bookmarks: { [key: string]: { ts: number } } }>({
```

Comment:
Can we create (and export) a type for `{ bookmarks: { [key: string]: { ts: number } } }`?
We should also use naming that is use else in the project (e.g. use `id` instead of `key`)